src/main/java/picard/arrays/CreateVerifyIDIntensityContaminationMetricsFile.java

```diff
@@ -0,0 +1,117 @@
+import htsjdk.samtools.util.IOUtil;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import picard.PicardException;
+import picard.cmdline.CommandLineProgram;
+import picard.cmdline.StandardOptionDefinitions;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A simple program to create a standard picard metrics file
+ * from the output of VerifyIDIntensity.
+ */
+@CommandLineProgramProperties(
+        summary = CreateVerifyIDIntensityContaminationMetricsFile.USAGE_DETAILS,
+        oneLineSummary = "Program to generate a picard metrics file from the output of the VerifyIDIntensity tool.",
+        programGroup = picard.cmdline.programgroups.GenotypingArraysProgramGroup.class
+)
+
+@DocumentedFeature
+public class CreateVerifyIDIntensityContaminationMetricsFile extends CommandLineProgram {
+    static final String USAGE_DETAILS =
+            "CreateVerifyIDIntensityContaminationMetricsFile takes an output file as generated by the VerifyIDIntensity tool and creates a picard metrics file. " +
+                    "VerifyIDIntensity <a href='https://genome.sph.umich.edu/wiki/VerifyIDintensity'>VerifyIDintensity</a> is a tool for " +
+                    "detecting and estimating sample contamination of Illumina genotyping array data." +
+                    "<h4>Usage example:</h4>" +
+                    "<pre>" +
+                    "java -jar picard.jar CreateVerifyIDIntensityContaminationMetricsFile \\<br />" +
+                    "      INPUT=VerifyIDIntensityOutput.txt \\<br />" +
+                    "      OUTPUT=outputBaseName" +
+                    "</pre>";
+
+    @Argument(shortName = StandardOptionDefinitions.INPUT_SHORT_NAME, doc = "The output of VerifyIDIntensity(typically captured stdout).")
+    public File INPUT;
+
+    @Argument(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Basename for the metrics file that will be written." +
+            " Resulting file will be <OUTPUT>." + FILE_EXTENSION)
+    public File OUTPUT;
+
+    public static final String FILE_EXTENSION = "verifyidintensity_metrics";
+
+    @Override
+    protected int doWork() {
+        IOUtil.assertFileIsReadable(INPUT);
+        final File metricsFile = new File(OUTPUT + "." + FILE_EXTENSION);
+        IOUtil.assertFileIsWritable(metricsFile);
+
+        final MetricsFile<VerifyIDIntensityContaminationMetrics,?> verifyIDIntensityContaminationMetricsFile = getMetricsFile();
+
+        final Pattern HEADER_PATTERN = Pattern.compile("^ID\\s+%Mix\\s+LLK\\s+LLK0\\s*$");
+        final Pattern DASHES_PATTERN = Pattern.compile("^[-]+$");
+        final Pattern DATA_PATTERN = Pattern.compile("^(\\d+)\\s+(\\d*\\.\\d+)\\s+(\\d+)\\s+(\\d+)\\s*$");
```

Comment:
Exponential notation not required for the double field?

Reply:
Not that I've seen.